### DISCUSSION TITLE
Announcing the upcoming general availability of MRTK3

### DISCUSSION THREAD

**Discussion by AMollis** [microsoft/MixedRealityToolkit-Unity#11597] [General]
![image](https://github.com/microsoft/MixedRealityToolkit-Unity/assets/36461279/a594e0d0-5282-4e3a-aace-0fa70456b338)

Microsoft has made a public announcement about the General Availability (GA) of MRTK3. We're pleased to announce that MRTK3 will reach General Availability in Q3 of this calendar year. You can read more at Robin's blog post:

https://techcommunity.microsoft.com/t5/mixed-reality-blog/announcing-the-upcoming-general-availability-of-microsoft-mixed/ba-p/3833710

**Comment by BertrandDeSaintBezier** [comment]
Long live the King

**Comment by Victor-x1** [comment]
Q3 is traditionally July 1 – September 30. That's a lot sooner than expected! :)

**Comment by XRWebland** [comment]
@AMollis this is really great news seeing the ongoing support
